samples/nrf9160/aws_fota/src/main.c

```diff
@@ -104,11 +153,6 @@ void mqtt_evt_handler(struct mqtt_client * const c,
 		return;
 	} else if (err < 0) {
 		printk("aws_fota_mqtt_evt_handler: Failed! %d\n", err);
-		printk("Disconnecting MQTT client...\n");
```

Comment:
Could you clarify, why do we remove this disconnect, apart from moving stuff around?

Reply:
This is a separated change I could remove it out. 

Reasoning behind removing this is to ensure that the sample handles whenever it needs to disconnect. the AWS Fota MQTT library should not just disconnect on error. If you think it should be moved to another change that's fair and just have this as a simple move change.

Reply:
The commit message could include some rationale behind this change, or it could be even split into a separate commit, but I don't have a strong push for that. Thanks for explaining.

Reply:
Moved it into a separate commit